test/test_core.py

```diff
@@ -1273,6 +1268,7 @@ def test_exceptions_minimal_runtime(self):
       self.set_setting('DISABLE_EXCEPTION_CATCHING', 0)
       self.do_run_from_file(test_file('core/test_exceptions.cpp'), test_file('core/test_exceptions_caught.out'))
 
+      self.set_setting('EXCEPTION_DEBUG')
```

Comment:
Was there a reason for this change?  (Maybe I did this, but I can't remember why).

Reply:
Yeah you did, but no, I don't know why.

Reply:
Looks like you discussed this above:
![image](https://user-images.githubusercontent.com/557590/228677858-98e60c07-f6a3-4bc9-8d98-ed344ad277a7.png)

Reply:
That person sounds very logical.  I can't argue with them.